### PR TITLE
Simple 1-axis arm support

### PR DESCRIPTION
**Pull request by BrownGenius**

### CODE REVIEW

**Review by wojtase1 — DISMISSED** [review]

File: src/main/java/frc/robot/commands/ArmCommand.java [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    m_arm.setArmBasePosition(m_defaultPosition);
+  }
+
+  @Override
+  public void execute() {
+    // Variable that will store the new calculated arm position
+    double newArmPosition;
+
+    // Determine the requested speed, but ignoring inputs near-zero (i.e. +/= m_deadband)
+    double currentShoulderSpeed = MathUtil.applyDeadband(m_baseSpeed.getAsDouble(), m_deadband);
```
Comment (loafdog) [inline src/main/java/frc/robot/commands/ArmCommand.java]: Maybe call this requested or new ShoulderSpeed? It isnt the current speed of the arm.
Also the word speed is throwing me off.  Maybe a different term/word like offset or target?

File: src/main/java/frc/robot/subsystems/Arm.java [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+   * the servo is flipped, the directions would be reversed. This module abstracts the hardware
+   * implementation details and follows the convention of 1.0 is up.
+   *
+   * @param position desired target position for arm base [-1.0 to 1.0]
+   */
+  public void setArmBasePosition(double position) {
+    // Note: This code doesn't validate the requested position.  If we don't want the arm to move
+    // within the entire range (e.g. due to physical constraints), we could clamp the position to
+    // the min/max values before setting the servo's position
+    m_armBaseServo.set(position);
```
Comment (loafdog) [inline src/main/java/frc/robot/subsystems/Arm.java:29]: docs say range is 0..1 not -1..1 ? https://github.wpilib.org/allwpilib/docs/release/java/edu/wpi/first/wpilibj/Servo.html#set(double)

File: src/main/java/frc/robot/commands/ArmCommand.java (hunk 1 above)
Comment (BrownGenius) [reply src/main/java/frc/robot/commands/ArmCommand.java]: I renamed it "newBasePositionDelta"

File: src/main/java/frc/robot/subsystems/Arm.java (hunk 2 above)
Comment (BrownGenius) [reply src/main/java/frc/robot/subsystems/Arm.java:29]: I changed it to 0.0 to 1.0.  Thanks for catching this!

**Review by loafdog — APPROVED** [review]

**Pull request merged**